site2/website/release-notes.md

```diff
@@ -797,6 +797,116 @@ For more context about this issue, see [[Pulsar Functions and IO] Cannot upgrade
+- Only auth_errors should log at error level. [#9325](https://github.com/apache/pulsar/pull/9325)
+
+#### Dependency upgrade
+- Upgrade Log4j2 version to 2.14.0 and replace legacy log4j dependency. [#8880](https://github.com/apache/pulsar/pull/8880)
+- Remove Boost::System runtime dependency. [#9498](https://github.com/apache/pulsar/pull/9498)
+- Upgrade commons-compress to 1.21. [#11345](https://github.com/apache/pulsar/pull/11345)
+- Upgrade Jetty to 9.4.43.v20210629. [#11660](https://github.com/apache/pulsar/pull/11660)
+- Add maven.restlet.org repository. [#13248](https://github.com/apache/pulsar/pull/13248)
+- Bump log4j to 2.15.0. [#13226](https://github.com/apache/pulsar/pull/13226)
+- Upgrade netty to 4.1.68.Final. [#13215](https://github.com/apache/pulsar/pull/13215)
+- Upgrade Zookeeper version to 3.5.9. [#12981](https://github.com/apache/pulsar/pull/12981)
+- Remove pulsar-dashboard from the publish process of images. [#12534](https://github.com/apache/pulsar/pull/12534)
+- Roll back to using Java 8 for docker images. [#12357](https://github.com/apache/pulsar/pull/12357)
+- Upgrade TestContainers to 1.15.1. [#9120](https://github.com/apache/pulsar/pull/9120)
+
+#### Client
+- [Java] Refactor seek to reuse common logic. [#9670](https://github.com/apache/pulsar/pull/9120)
+- [Java] Process partitioned-topic messages on different listener-threads. [#10017](https://github.com/apache/pulsar/pull/10017)
+- [C++] Use sendRequestWithId to add timeout to hasMessageAvailable. [#11600](https://github.com/apache/pulsar/pull/11600)
+- [Python] Fix fields that are ignoring the required key argument. [#11508](https://github.com/apache/pulsar/pull/11508)
+- [Java] Fix Consumer listener does not respect receiver queue size. [#11455](https://github.com/apache/pulsar/pull/11455)
+- [Java] Add a optional params scope for pulsar oauth2 client. [#11931](https://github.com/apache/pulsar/pull/11931)
+- [Java] fix seek at batchIndex level receive duplicated messages. [#11826](https://github.com/apache/pulsar/pull/11826)
+- [Java] Fixed race condition on multi-topic consumer. [#11764](https://github.com/apache/pulsar/pull/11764)
+- [Java] remove consumer reference from PulsarClient on subscription failure. [#11758](https://github.com/apache/pulsar/pull/11758)
+- [Java] Fixed block forever bug in Consumer.batchReceive. [#11691](https://github.com/apache/pulsar/pull/11691)
+- [Java] Send CloseProducer on timeout. [#13161](https://github.com/apache/pulsar/pull/13161)
+- [Pulsar perf] Fix log level config for pulsar-admin, pulsar-client and pulsar-perf. [#12915](https://github.com/apache/pulsar/pull/12915)
+- [Java] Remove invalid call to Thread.currentThread().interrupt(). [#12652](https://github.com/apache/pulsar/pull/12652)
+- [C++] Removed usages of boost::regex. [#9533](https://github.com/apache/pulsar/pull/9533)
```

Comment:
can you get them organized to improve readability? something like:

```
- [Java] xx
- [Java] xx
...
- [C++] xx
- [C++] xx
...
- [Python] xx
- [Python] xx
...
```

Reply:
Ok